production/helm/loki/templates/tokengen/job-tokengen.yaml

```diff
@@ -75,7 +75,7 @@ spec:
           image: {{ include "loki.kubectlImage" . }}
           imagePullPolicy: {{ .Values.kubectlImage.pullPolicy }}
           command:
-            - /bin/bash
+            - ! test -f /shared/admin-token && /bin/bash
```

Comment:
Trevor just explained it to me. The init container job would not create the file if it's already in the object storage. Could you add this as a comment?